internal/pkg/cli/secret_init_test.go

```diff
@@ -434,36 +434,6 @@ func TestSecretInitOpts_Execute(t *testing.T) {
 				}, nil)
 			},
 		},
-		"error out if received other errors": {
```

Comment:
Would it make sense to add a test here that includes calling execute with an overwrite flag to test the update functionality?

Reply:
Good call! Updated!